airflow/utils/dag_processing.py

```diff
@@ -465,6 +465,18 @@ def _split_path(file_path):
         results.reverse()
         return results
 
+    def _get_log_directory(self):
+        """
+        Log output from processing DAGs for the current day should go into
+        this directory.
+
+        :return: the path to the corresponding log directory
+        :rtype: unicode
+        """
+        now = datetime.now()
+        return os.path.join(self._child_process_log_directory,
+            now.strftime("%Y-%m-%d"))
```

Comment:
This path should be coupled with the logic (factored out int a function) that creates the directory to ensure that if the format changes in one place it will change it in the other.